index.js

```diff
@@ -6,9 +6,11 @@ const macos = require('./lib/macos');
 const linux = require('./lib/linux');
 const win = require('./lib/win');
 
-module.exports = iterable => {
+module.exports = (iterable, opts) => {
 	iterable = typeof iterable === 'string' ? [iterable] : iterable;
-	const paths = globby.sync(Array.from(iterable).map(String), {nonull: true})
+	iterable = Array.from(iterable).map(String);
```

Comment:
I would just do `iterable = Array.from(iterable = typeof iterable === 'string' ? [iterable] : iterable).map(String)`. This line seems moot.

Reply:
How about this one, it should be even safer because it resolves any type of literal, array or object into a flat array, and then stringifies it:

```javascript
iterable = [].concat(iterable).map(String);
```

Reply:
@mofux `.concat` doesn't accept an iterable.

Reply:
@sindresorhus You're right, I've just tested it, still shocked. What a pity. Shall we go with @kevva suggestion then?

```javascript
iterable = Array.from(typeof iterable === 'string' ? [iterable] : iterable).map(String);
```

Reply:
> What a pity. Shall we go with @kevva suggestion then?

Yes